.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,5 +1,5 @@
-### What are the changes introduced in this pull request?
+#### What are the changes introduced in this pull request?
 
-### What is the thinking behind these changes?
+#### What is the thinking behind these changes?
```

Comment:
In similar spirit, how about something like 'Considerations taken when implementing this change?' The current phrasing almost sounds accusatory ;)

Reply:
sounds good!